<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConfig</class>
 <widget class="QDialog" name="FormConfig">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Account Configuration Test</string>
  </property>
  <widget class="QToolButton" name="TButtonConfigDbSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Save a valid configuration.</string>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/PNG/48x48/Filesaveas-48x48.png</normaloff>:/images/PNG/48x48/Filesaveas-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="GBoxConfigDb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>480</width>
     <height>330</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Database Configuration Account.</string>
   </property>
   <widget class="QLabel" name="ConfigLBDbHost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Database Address:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbPort">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the connection port to the database!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Database Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the database name!&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Database User:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbUser">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the database connection user!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbPass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbPass">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the database connection password!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonTestConn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Test connection.</string>
    </property>
    <property name="text">
     <string>Test Conn...</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/PNG/120x120/120px-Gtk-refresh.png</normaloff>:/images/PNG/120x120/120px-Gtk-refresh.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ConfigLBDbTestResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Result:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbTestResult">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>370</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Shows error or success messages!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfigLEDbHost">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the database connection address!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <zorder>ConfigLEDbName</zorder>
   <zorder>ConfigLBDbHost</zorder>
   <zorder>ConfigLBDbPort</zorder>
   <zorder>ConfigLBDbName</zorder>
   <zorder>ConfigLBDbUser</zorder>
   <zorder>ConfigLEDbUser</zorder>
   <zorder>ConfigLBDbPass</zorder>
   <zorder>ConfigLEDbPass</zorder>
   <zorder>TButtonTestConn</zorder>
   <zorder>ConfigLBDbTestResult</zorder>
   <zorder>ConfigLEDbTestResult</zorder>
   <zorder>ConfigLEDbHost</zorder>
   <zorder>ConfigLEDbPort</zorder>
  </widget>
  <widget class="QToolButton" name="TButtonConfigDbExit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Close the window.
</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/PNG/48x48/Exithome-48x48.png</normaloff>:/images/PNG/48x48/Exithome-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
